lib/scope.js

```diff
@@ -2248,13 +2446,15 @@ module.exports = {
       if ( password === undefined ) {
         await scope.addToReport( scope, { type: `error`, value: password_msg })
       }
+
       expect( email, email_msg ).to.not.equal( undefined );
       expect( password, password_msg ).to.not.equal( undefined );
 
       // Try to log in
       await scope.page.type( `#email`, email );
       await scope.page.type( `#password`, password );
-      await scope.page.$eval( `button[name="submit"]`, (elem) => { return elem.click(); });
+      let elem = await scope.page.$( `button[name="submit"]` );
+      await scope.tapElement( scope, elem );
```

Comment:
Would we pass selectors to `.tapElement()` instead? I'd have to look more closely at the code to see if that's feasible, but it might work. Not for this PR, but we could make it an issue.

Reply:
#592

Reply:
We'd want to check if it's actually slowing things down first, but an issue is fine